ajax/libs/bootstrap-markdown-editor/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "bootstrap-markdown-editor",
+  "filename": "js/bootstrap-markdown-editor.js",
+  "description": "Markdown editor for Bootstrap with preview, image upload support, shortcuts and other features.",
+  "version": "2.0.2",
+  "author": "Ignacio de Tomás <nacho@inacho.es>",
+  "homepage": "https://inacho.github.io/bootstrap-markdown-editor/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/inacho/bootstrap-markdown-editor.git"
+  },
+  "keywords": [
+    "markdown",
+    "bootstrap",
+    "editor",
+    "ace"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/inacho/bootstrap-markdown-editor.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "src/bootstrap-markdown-editor*.+(js|map)"
+        ]
+      }
```

Comment:
@PeterDaveHello would you please tell us abut the [`bootstrap-markdown-editor.js`](https://github.com/inacho/bootstrap-markdown-editor/blob/v2.0.2/src/bootstrap-markdown-editor.js) which is non-minified file under `src`, can it be added?

Because this file can be used in front-end.
But the minified files are under `dist`, the file structure is as the following:
```
dist
 ├── css
 │   └── bootstrap-markdown-editor.css
 └── js
     └── bootstrap-markdown-editor.js
```

If we add the file under src, should the file structure as the following?
```
├── 2.0.2
│   ├── css
│   │   └── bootstrap-markdown-editor.css
│   ├── js
│   │   └── bootstrap-markdown-editor.js
│   └── src
│       └── bootstrap-markdown-editor.js
└── package.json
```

Many thanks.

PS. If the file under src should be added, the file should be added in this PR.